python_docs_theme/static/pydoctheme.css

```diff
@@ -191,3 +200,228 @@ div.footer a:hover {
 dl > dt span ~ em {
     font-family: monospace, sans-serif;
 }
+
+@media (max-width: 900px) {
+    /* Body layout */
+    div.body {
+        min-width: 100%;
+        padding: 0;
+        font-size: 0.875rem;
+    }
+    div.bodywrapper {
+        margin: 0;
+    }
+    /* Typography */
+    div.body h1, div.body h2, div.body h3, div.body h4, div.body h5, div.body p {
+        font-family: SourceSansProRegular, Arial, sans-serif;
+    }
+    div.body h1 {
+        font-size: 1.625rem;
+    }
+    div.body h2 {
+        font-size: 1.25rem;
+    }
+    div.body h3, div.body h4, div.body h5 {
+        font-size: 1rem;
+    }
+    /* Remove sidebar and top related bar */
+    div.related, .sphinxsidebar {
+        display: none;
+    }
+    /* Top navigation bar */
+    .mobile-nav {
+        display: block;
+        padding: 0 1rem;
+        height: 40px;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: white;
+        box-shadow: rgba(0, 0, 0, 0.25) 0 0 2px 0;
+        z-index: 5;
+    }
+    .nav-content {
+        padding-left: 40px;
+        height: 40px;
+        display: flex;
+        align-items: center;
+    }
+    .nav-content img {
+        flex: 0 0 auto;
+        margin-right: 0.7rem;
+    }
+    .version_switcher_placeholder {
+        margin-right: 1rem;
+    }
+    .version_switcher_placeholder select {
+        height: 24px;
+    }
+    .nav-content #searchbox {
+        flex: 5 1 auto;
+    }
+    .nav-content .search {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        box-sizing: border-box;
+        padding: 0 0 0 2px;
+        border: 1px solid #a9a9a9;
+        height: 24px;
+    }
+    .nav-content .search:hover {
+        box-shadow: 0 1px 6px 0 rgba(32,33,36,0.28);
+        border-color: rgba(223,225,229,0);
+    }
+    .nav-content .search input[type=text] {
+        border: 0;
+        outline: 0;
+        box-shadow: none;
+        width: 40px;
+        height: 100%;
+        flex: 1 1 auto;
+    }
+    .nav-content .search input[type=submit] {
+        height: 100%;
+        border: 0;
+        box-shadow: none;
+        outline: 1px solid #999;
+    }
+    .nav-content .search label {
+        border: 0;
+        clip: rect(0 0 0 0);
+        height: 1px;
+        margin: -1px;
+        overflow: hidden;
+        padding: 0;
+        position: absolute !important;
```

Comment:
Please try to not use `!important`

Reply:
Replaced visually-hidden label for accessibility by `aria-label`s